scripts/sct_maths.py

```diff
@@ -375,19 +370,19 @@ def main(args=None):
         # input 1 = from flag -i --> im
         # input 2 = from flag -mi
         im_2 = Image(arguments.mi)
-        compute_similarity(im.data, im_2.data, fname_out, metric='mi', verbose=verbose)
+        compute_similarity(im, im_2, fname_out, metric='mi', metric_full='Mutual information', verbose=verbose)
```

Comment:
why changing the object from np.array to Image if not changing the current input?
https://github.com/neuropoly/spinalcordtoolbox/blob/149afde200700707324fca68b1d0d248d5ff0afe/spinalcordtoolbox/math.py#L190

Reply:
ah sorry-- i'm just seeing that there is another `compute_similarity` defined here-- nevermind.

Reply:
Yes the API was doing too much, only kept the computation in the API and moved the rest in the script.